packages/service-clients/odsp-client/src/odspClient.ts

```diff
@@ -65,6 +70,37 @@ async function getWebsocketToken(
 	return tokenResponse;
 }
 
+/**
+ * Default feature gates.
+ * These values will only be used if the feature gate is not already set by the supplied config provider.
+ */
+const odspClientFeatureGates = {
+	// None yet
```

Comment:
@andre4i - we don't need `ForceWriteConnection` here, right?

Reply:
that's right